### PR TITLE
[PhongCH] custom

### PR DESCRIPTION
**Pull request by phongchauchp**

### CODE REVIEW

File: app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/mysong/subfragment/album/AlbumAdapter.java [hunk 1]
```diff
@@ -91,8 +94,17 @@ private void bind(int position) {
...
+
+            if (nameAlbum.length() >= MAX_LENGTH) {
+                String subSongs = nameAlbum.substring(MIN_LENGTH, MAX_LENGTH) + "...";
+                mTxtAlbum.setText(subSongs);
+            } else {
+                mTxtAlbum.setText(nameAlbum);
+            }
             mTxtArtist.setText(nameArtist);
         }
+        private void setNameAlbum(int position) {
```
Comment (daolq3012) [inline app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/mysong/subfragment/album/AlbumAdapter.java]: WARNING: 'position' hides a field.

File: app/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -15,4 +16,5 @@
     <color name="color_deep_purple">#1b0143</color>
     <color name="color_little_purple">#5c2ba9</color>
     <color name="color_dark_opacity">#64000000</color>
+    <color name="color_mini_opacity">#b4000000</color>
```
Comment (daolq3012) [inline app/src/main/res/values/colors.xml]: WARNING: [UnusedResources] The resource `R.color.color_mini_opacity` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/mysong/subfragment/album/AlbumAdapter.java [hunk 3]
```diff
@@ -88,11 +91,19 @@ private void bind(int position) {
...
             this.position = position;
             Bitmap image = mAlbums.get(position).getBmAlbum();
-            String nameAlbum = mAlbums.get(position).getName();
             String nameArtist = mAlbums.get(position).getNameArtist();
             mImageAlbum.setImageBitmap(image);
-            mTxtAlbum.setText(nameAlbum);
             mTxtArtist.setText(nameArtist);
+            setNameAlbum(position);
+        }
+        private void setNameAlbum(int position) {
```
Comment (daolq3012) [inline app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/mysong/subfragment/album/AlbumAdapter.java]: WARNING: 'position' hides a field.

**Comment by tranminhtri911** [comment]
Ok

**Pull request merged**